docs/config/services/deployments/production.js

```diff
@@ -1,16 +1,30 @@
 'use strict';
 
 var versionInfo = require('../../../../lib/versions/version-info');
-var cdnUrl = '//ajax.googleapis.com/ajax/libs/angularjs/' + versionInfo.cdnVersion;
+
+var googleCdnUrl = '//ajax.googleapis.com/ajax/libs/angularjs/';
+var angularCodeUrl = '//code.angularjs.org/';
+
+var cdnUrl = googleCdnUrl + versionInfo.cdnVersion;
+
+// The plnkr examples must use the code.angularjs.org repo for the snapshot,
+// and the cdn for the tagged version and, if the build is not tagged, the currentVersion.
+//
+// The currentVersion may not be available on the cdn (e.g. if built locally, or hasn't been pushed
+// yet). This will lead to a 404, but this is preferable to loading a version with which the example
+// might not work (possibly in subtle ways).
+var examplesCdnUrl = versionInfo.isSnapshhot ?
+  (angularCodeUrl + 'snapshot') :
+  (googleCdnUrl + (versionInfo.version || versionInfo.currentVersion));
```

Comment:
I guess if the `versionInfo.version` is not available then we are doing a local non-snapshot build. In which case the plunkers are going to be useless. Perhaps we should default to the snapshot though in that case?